services/QuillLMS/app/models/unit_template.rb

```diff
@@ -140,6 +140,43 @@ def self.delete_all_caches
     end
   end
 
+  def self.student_counts_for_previously_assigned_activity(unit=nil, classrooms=[])
+    classrooms&.map do |classroom|
```

Comment:
I don't think you need the safe navigation operator since you're explicitly asking for `[]` if `classrooms` is falsy, right?

Reply:
yeah, you're right-- thanks for catching that!